README.md

```diff
@@ -46,25 +46,25 @@ Download and verify the above files from the below table:
-`query_doc_ids_dev.txt` | 134 M | `0b1ef659127339f71e70da28d63e513c` | [[GCS](https://storage.googleapis.com/duobert_git/query_doc_ids_dev.txt)] [[Dropbox](https://www.dropbox.com/s/er9ider35t1hwhv/query_doc_ids_dev.txt)]
-`query_doc_ids_test.txt` | 131 M | `a857664b0cb47219266a6edeb7eb4b6c` | [[GCS](https://storage.googleapis.com/duobert_git/query_doc_ids_test.txt)] [[Dropbox](https://www.dropbox.com/s/hqk3cs82cwfmaw3/query_doc_ids_test.txt)]
-`queries.dev.small.tsv` | 283 KB | `41e980d881317a4a323129d482e9f5e5` | [[GCS](https://storage.googleapis.com/duobert_git/queries.dev.small.tsv)] [[Dropbox](https://www.dropbox.com/s/hq6xjhswiz60siu/queries.dev.small.tsv)]
-`queries.eval.small.tsv` | 274 KB | `bafaf0b9eb23503d2a5948709f34fc3a` | [[GCS](https://storage.googleapis.com/duobert_git/queries.eval.small.tsv)] [[Dropbox](https://www.dropbox.com/s/kjf4bjk1vygipcc/queries.eval.small.tsv)]
-`qrels.dev.small.tsv` | 140 KB| `38a80559a561707ac2ec0f150ecd1e8a` | [[GCS](https://storage.googleapis.com/duobert_git/qrels.dev.small.tsv)] [[Dropbox](https://www.dropbox.com/s/5t6e2225rt6ikym/qrels.dev.small.tsv)]
-`collection.tar.gz` | 987 MB | `87dd01826da3e2ad45447ba5af577628` | [[GCS](https://storage.googleapis.com/duobert_git/collection.tar.gz)] [[Dropbox](https://www.dropbox.com/s/m1n2wf80l1lb9j1/collection.tar.gz)]
-`triples.train.small.tar.gz` | 7.4 GB | `c13bf99ff23ca691105ad12eab837f84` | [[GCS](https://storage.googleapis.com/duobert_git/triples.train.small.tar.gz)] [[Dropbox](https://www.dropbox.com/s/6r4a8hpcgq0szep/triples.train.small.tar.gz)]
+`bert-large-msmarco-pretrained-only.zip` |  3.44 GB | `88f1d0bd351058b1da1eb49b60c2e750` | [[Dropbox](https://www.dropbox.com/s/nvqs8qk7q63qr0s/bert-large-msmarco-pretrained-only.zip?dl=1)]
+`monobert-large-msmarco-pretrained-and-finetuned.zip` | 3.42 GB | `db201b6433b3e605201746bda6b7723b` | [[Dropbox](https://www.dropbox.com/s/fhy7vf5488muz9u/monobert-large-msmarco-pretrained-and-finetuned.zip?dl=1)]
+`duobert-large-msmarco-pretrained-and-finetuned.zip` | 3.43 GB | `dcae7441103ae8241f16df743b75337b` | [[Dropbox](https://www.dropbox.com/s/kxd8fitk4ax1hb5/duobert-large-msmarco-pretrained-and-finetuned.zip?dl=1)]
+`run.bm25.dev.small.tsv.gz` | 44 MB | `0a7802ab41999161339087186dda4145` | [[Dropbox](https://www.dropbox.com/s/5pqpcnlzlib2b3a/run.bm25.dev.small.tsv.gz?dl=1)]
+`run.bm25.test.small.tsv.gz` | 43 MB | `1ea465405f6a2467cb62015454bc88c7` | [[Dropbox](https://www.dropbox.com/s/6fzxajh79dkw8s1/run.bm25.test.small.tsv.gz?dl=1)]
+`run.monobert.dev.small.tsv.gz` | 44 MB | `dee6065e7177facb7c740f607e40ac63` | [[Dropbox](https://www.dropbox.com/s/h5kiff0ofn3djvf/run.monobert.dev.small.tsv.gz?dl=1)]
+`run.monobert.test.small.tsv.gz` | 43 MB | `f0e16234351a0a81d83f188e72662fbd` | [[Dropbox](https://www.dropbox.com/s/ctccble07k7lvlc/run.monobert.test.small.tsv.gz?dl=1)]
```

Comment:
remove `?dl=1`?

Reply:
`?dl=1` is necessary for our check script to work - because otherwise Dropbox takes you to a landing page.

Even if you download in the browser, without `?dl=1` it's annoying because you get to a landing page, and then you have to click another link to actually download. With `?dl=1` we get direct download.

Try it out each and see what you think?

Reply:
Ah, got it, ok!